internal/cypress/v1/config.go

```diff
@@ -264,7 +266,25 @@ func (p *Project) Validate() error {
 	return nil
 }
 
-func shardSuites(rootDir string, suites []Suite, ccy int) ([]Suite, error) {
+// excludeSauceIgnorePatterns excludes any match with sauceignore content.
+// If loading and parsing the sauceignore content fails, no filtering is applied.
+func excludeSauceIgnorePatterns(files []string, sauceignoreFile string) []string {
+	matcher, err := sauceignore.NewMatcherFromFile(sauceignoreFile)
+	if err != nil {
+		log.Warn().Err(err).Msgf("an error occurred when filtering specs with %s. No filter will be applied", sauceignoreFile)
```

Comment:
It's fine to start log messages with a capital. Just not in the actual `error` 👍 
```suggestion
		log.Warn().Err(err).Msgf("An error occurred when filtering specs with %s. No filter will be applied", sauceignoreFile)
```